cmd/webhook/webhook.go

```diff
@@ -160,7 +165,11 @@ func patchContainerForPinning(cpuAnnotation types.CPUAnnotation, patchList []pat
 	var patchItem patch
 
 	glog.V(2).Infof("Adding CPU pinning patches")
-
+	for _, volumeMount := range c.VolumeMounts{
```

Comment:
Log should be written only if we proceed to patching so the logging should be after this.